lib/main.dart

```diff
@@ -62,11 +62,22 @@ class _MyAppState extends State<MyApp> {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-      home: Scaffold(
-        appBar: AppBar(
-          title: const Text('Plugin example app'),
+        home: Scaffold(
+      appBar: AppBar(
```

Comment:
let's try to keep it clean in the repo by formating structures to match their hierarchy, this can be left as it is, but in future keep it clean